projects/ion/src/lib/indicator/indicator.component.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import {
+  render,
+  RenderResult,
+  screen,
+  fireEvent,
+} from '@testing-library/angular';
+import { ButtonModule } from '../button/button.module';
+import { IonModalComponent } from '../modal/component/modal.component';
+import { TooltipModule } from '../tooltip/tooltip.module';
+import { BodyMockComponent } from './../card/mock/body-mock.component';
+import { IonIndicatorComponent } from './indicator.component';
+import {
+  buttonEmitterConfig,
+  buttonModalConfig,
+  buttonRedirectConfig,
+} from './mocks/indicator-button-config';
+import { IonIndicatorButtonConfiguration } from '../core/types/indicator';
+
+interface IonIndicatorProps {
```

Comment:
You could set this interface inside the component or even on core folder, so it could be used for anyone who is implementing a indicator and needs to use this type as a guide.